plugins/config_manager.py

```diff
@@ -167,7 +169,7 @@ def add_repo(self):
             else:
                 # just url to repo, create .repo file on our own
                 repoid = sanitize_url_to_fs(url)
-                reponame = 'created by dnf config-manager from %s' % url
+                reponame = 'created by {0} config-manager from {1}'.format(dnf.util.MAIN_PROG, url)
```

Comment:
Out of curiosity, why use named placeholder on the previous string and numbered placeholders on this one?

Also, this string isn't translated.. do we want to translate it perhaps?

Reply:
I believe that we don't want to translated. This information is stored in .repo files and names should be same generated by same command. Is something wrong in substitution that I used? Because the string is not translated the substitution format is not so important.

Reply:
I see, you're right, we don't want this translated I think.

And nothing wrong about it, just looking at this commit the formatting placeholders are inconsistent, but I don't really mind.

Reply:
Ok I changed it to `'created by {} config-manager from {}'.format(dnf.util.MAIN_PROG, url)`